Update/kake_032.txt

```diff
@@ -421,7 +421,7 @@ void main()
 //r涙さえ出てくるような毒臭を吸い込むまいと息を止めながら、換気装置らしきものはないかと周囲に目を向ける。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "涙さえ出てくるような毒臭を吸い込むまいと息を止めながら、換気装置らしきものはないかと周囲に目を向ける。",
-			NULL, "I held my breath to avoid inhaling the odor that was starting to make tears come out, and looked around for any signs of ventilation.", Line_Normal);
+			NULL, "I held my breath to avoid inhaling the odor that was tearing my eyes up, and looked around for any signs of ventilation.", Line_Normal);
```

Comment:
```suggestion
			NULL, "I held my breath to avoid inhaling the odor that was making me teary-eyed, and looked around for any signs of ventilation.", Line_Normal);
```